### PR TITLE
Allow user secrets in dotnet-isolated projects

### PR DESCRIPTION
**Pull request by anthonychu**

### CODE REVIEW

**Review by ankitkumarr — APPROVED** [review]

**Comment by gzuber** [comment]
This looks good! Is there a way to add this case to [the existing user secrets tests](https://github.com/Azure/azure-functions-core-tools/blob/v3.x/test/Azure.Functions.Cli.Tests/E2E/StartTests.cs#L532-L604)? @anthonychu I'd be happy to work with you on it or push a commit just let me know

File: src/Azure.Functions.Cli/Helpers/ProjectHelpers.cs [hunk 1]
```diff
@@ -40,7 +40,7 @@ public static string FindProjectFile(string path, LoggingFilterHelper loggingFil
                 {
                     foreach (FileInfo file in projectFiles)
                     {
-                        if (string.Equals(projectFiles[0].Name, Constants.ExtenstionsCsProjFile, StringComparison.OrdinalIgnoreCase)) continue;
```
Comment (fabiocav) [inline src/Azure.Functions.Cli/Helpers/ProjectHelpers.cs:43]: 😮

**Review by fabiocav — APPROVED** [review]

**Comment by anthonychu** [comment]
@gzuber At the time I don't think we had all the pieces in place to write a test for dotnet-isolated. We probably do now. Let me know if you want to help me with this.

**Comment by surgupta-msft** [comment]
Merged these changes as part of this PR - https://github.com/Azure/azure-functions-core-tools/pull/3052

**Pull request closed without merging**